exchange/adapter_util.go

```diff
@@ -127,7 +77,9 @@ func GetActiveBidders(infos config.BidderInfos) map[string]openrtb_ext.BidderNam
 
 // GetDisabledBiddersErrorMessages returns a map of error messages for disabled bidders.
 func GetDisabledBiddersErrorMessages(infos config.BidderInfos) map[string]string {
-	disabledBidders := make(map[string]string)
+	disabledBidders := map[string]string{
```

Comment:
From what I understand, Lifestreet being an exclusive legacy adapter was only supported on the `/auction` endpoint, correct? If that is the case, then do we need to add it in the `disabledAdapters` map here? Because this map is only used by openrtb2 endpoints and they never supported `lifestreet` to start with

Reply:
All legacy adapters had a wrapper that would allow them to be used with the openrtb2 endpoint. So yes we do.